EDDI/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # CHANGE LOG
 
+### 2.4.2.1
```

Comment:
Important: 2.4.3 please or the auto-updater will break!

Reply:
We want to stay with 2.4 as long as Elite is at 2.4, correct? If we go to 2.4.3 for a minor bugfix and continue to release regular updates, it won't be long before we hit 2.4.9...

I'll change it, but we need to think about changing what versioning we can support in the future or else abandon the notion that our version numbering is linked to Elite's versioning.

Reply:
Added commit to revise the Changelog & advance develop to 2.4.3.

Reply:
My first thought would be to stay with what jgm had done in the past: if we describe the version string as `major.minor.patch`, keep the major and minor parts in sync with the game but let the patch increase as necessary, past 10 if we have to. For example the 2.0 series went up to 2.0.13.

It is after all, a dot-separated tuple, not a decimal.

Reply:
Sounds reasonable.